### PR TITLE
Format Inside-Out I/O Source for Doxygen

### PR DESCRIPTION
**Pull request by danintel**
Before running Doxygen, this reformats the comments so that they can
be parsed by Doxygen, the automatic document from source generator.

For this first phase, only Inside-Out I/O is formatted.
Other source will be reformatted later.

Signed-off-by: danintel <daniel.anderson@intel.com>

### CODE REVIEW

File: common/sgx_workload/iohandler/file_io.cpp [hunk 1]
```diff
@@ -26,58 +26,89 @@
 #define MAX_FILE_SIZE 1024
 #define MAX_IO_RESULT_SIZE 128
 
-// Returns iohandler id corresponding to iohandler name
+
+/**
+ * Get the I/O handler ID corresponding to IoHandler handler_name.
+ *
+ * @param   handlerName Name of handler
```
Comment (manojgop) [inline common/sgx_workload/iohandler/file_io.cpp]: handlerName  -> handler_name

File: common/sgx_workload/iohandler/file_io.cpp [hunk 2]
```diff
@@ -128,12 +167,17 @@ uint32_t FileIoExecutor::FileTell(uint8_t *result, size_t result_size,
...
-   result_size - Maximum size of the result buffer
-*/
-uint32_t FileIoExecutor::FileSeek(size_t position, uint8_t *result, size_t result_size) {
+/**
+ * Moves the file position the file to the given position and
+ * updates the status in result buffer.
+ *
+ * @result      Status of file seek operation (0 is success,
+ *              non-0 is failure)
+ * @result_size Maximum size of the result buffer
```
Comment (manojgop) [inline common/sgx_workload/iohandler/file_io.cpp]: @param position
@param result
@param result_size

File: common/sgx_workload/iohandler/file_io.cpp [hunk 3]
```diff
@@ -144,12 +188,15 @@ uint32_t FileIoExecutor::FileSeek(size_t position, uint8_t *result, size_t resul
...
-   result - status of file delete operation
-   result_size - Maximum size of the result buffer
-*/
 
+/**
+ * Deletes the file whose name is stored in the FileIoExecutor instance.
+ *
+ * @result      Status of file delete operation (0 is success,
+ *              non-0 is failure)
+ * @result_size Maximum size of the result buffer
```
Comment (manojgop) [inline common/sgx_workload/iohandler/file_io.cpp]: @param result
@param result_size

File: common/sgx_workload/iohandler/file_io_wrapper.cpp [hunk 4]
```diff
@@ -18,6 +18,13 @@
 #include "file_io.h"
 #include "file_io_wrapper.h"
 
+/**
+ * Read a file named file_name and return the contents in a string.
+ * Return an empty string ("") on failure.
+ *
+ * @param param file_name Name of the file to be read
```
Comment (manojgop) [inline common/sgx_workload/iohandler/file_io_wrapper.cpp]: @param param file_name -> @param file_name

File: tc/sgx/trusted_worker_manager/enclave/iohandler_enclave.cpp [hunk 5]
```diff
@@ -22,23 +22,26 @@
...
-  inBufSize - max size of buffer having input data
-  outBuf - placeholder for output data
-  outBufSize - max size of buffer having output data
+
+/**
+ * Executes the given I/O command by invoking ocall and stores the result of
+ * I/O execution in result buffer.
+ *
+ * @param handlerId  Identifier for iohandler
+ * @param command    IO command that needs to be executed outside enclave
```
Comment (manojgop) [inline tc/sgx/trusted_worker_manager/enclave/iohandler_enclave.cpp]: @param commandSize  size of the IO command

Comment (danintel) [reply tc/sgx/trusted_worker_manager/enclave/iohandler_enclave.cpp]: Fixed

File: common/sgx_workload/iohandler/file_io_wrapper.cpp (hunk 4 above)
Comment (danintel) [reply common/sgx_workload/iohandler/file_io_wrapper.cpp]: Fixed

File: common/sgx_workload/iohandler/file_io.cpp (hunk 3 above)
Comment (danintel) [reply common/sgx_workload/iohandler/file_io.cpp]: Fixed

File: common/sgx_workload/iohandler/file_io.cpp (hunk 2 above)
Comment (danintel) [reply common/sgx_workload/iohandler/file_io.cpp]: Fixed

**Pull request merged**